projects/vdk-core/src/vdk/internal/builtin_plugins/run/data_job.py

```diff
@@ -54,6 +54,32 @@ def new_datajob(
         return DataJob(data_job_directory, core_context, name)
 
 
+def save_step_result(context: JobContext, result: StepResult):
+    step_results = context.core_context.state.get("step_results")
```

Comment:
I belive StateStore was using StoreKey as type and not string . But I assume this is for prototyping urposes.

Reply:
Fair point, I'll add some new values to that ExecutionStoreKeys class and use them instead.